libtrace/src/readtrace.cpp

```diff
@@ -18,6 +18,8 @@ void print_all(const char *f) {
 
   TraceContainerReader t(f);
 
+  std::cout << t.get_meta()->DebugString() << std::endl;
```

Comment:
Looks like you overlooked this debugging output.

Reply:
Ah no I just forgot to describe it. It also dumps the meta frame before all the "real" frames, which I think makes sense for the `readtrace` utility, similar to https://github.com/BinaryAnalysisPlatform/bap-frames/blob/97bad46824fc1fcc2e3cd6a7b1f308a74b15e73d/test/tracedump.ml#L29-L33. It's just protobuf calling this method `DebugString()`.